files/system/etc/sysconfig/chronyd

```diff
@@ -1,3 +1,22 @@
-# Command-line options for chronyd
-# https://www.mankier.com/8/chronyd
-OPTIONS="-F 1"
\ No newline at end of file
+# Copyright © 2014-2024 GrapheneOS
+
```

Comment:
a license declaration for 2 characters seems excessive :smile:

Reply:
Yeah... perhaps.